pennylane/_qubit_device.py

```diff
@@ -38,11 +38,15 @@ class QubitDevice(Device):
             expectation values of observables. Defaults to 1000 if not specified.
     """
     #pylint: disable=too-many-public-methods
-    def __init__(self, wires=1, shots=1000, analytic=True):
+    def __init__(self, wires=1, shots=1000, analytic=True, rotate_basis=True):
```

Comment:
As per our discussion, ``rotate_basis`` was added such that devices that natively support expectation values do not rotate the circuit.

However, currently, I would argue that from the set of devices that are supported, neither are such devices. As this change does not seem to be too complicated, it might be something that would be worth adding, once there was a need indeed for this functionality.

Reply:
Yep, sounds good :+1: